applications/GeoMechanicsApplication/tests/test_partial_saturation.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.GeoMechanicsApplication.geomechanics_analysis as analysis
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import test_helper
+
+class KratosGeoMechanicsPartialSaturation(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def __test_saturated_below_phreatic_level_pw(self, test_name):
+        n_stages = 2
+
+        # get the parameter file names for all stages
+        file_path = test_helper.get_file_path(os.path.join('.', 'test_partially_saturated', test_name))
```

Comment:
No need to explicitly include `.` here, since it is a relative path anyway.
```suggestion
        file_path = test_helper.get_file_path(os.path.join('test_partially_saturated', test_name))
```

Reply:
done.